include/myst/kernel.h

```diff
@@ -95,6 +95,9 @@ typedef struct myst_kernel_args
     /* The number of threads that can be created (including the main thread) */
     size_t max_threads;
 
+    /* The tid/pid of the main thread passed from the host */
+    pid_t target_tid;
```

Comment:
This applies to all ethreads. So the comment about just "main thread" is not accurate.